server/src/ZetaSqlWrapper.ts

```diff
@@ -222,9 +245,7 @@ export class ZetaSqlWrapper {
       if (refModel) {
         const analyzeResult = await this.analyzeTableInternal(refModel.originalFilePath);
         if (analyzeResult.isOk()) {
-          table.columns = analyzeResult.value.resolvedStatement?.resolvedQueryStmtNode?.outputColumnList
-            .filter(c => c.column !== null)
-            .map(c => ZetaSqlWrapper.createSimpleColumn(c.name, c.column?.type ?? null));
+          this.fillTableWithAnalyzeResponse(table, analyzeResult.value);
```

Comment:
I still feel like we're doing it twice. Why we can't delete this call from here? Looks like we should do the same when we call `this.analyzeTableInternal(refModel.originalFilePath);`.